.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
@oheimburger whats this file? To be removed?

Reply:
No idea how it got there. Maybe somehow from VS Code? We can remove it